### ISSUE TITLE
JAVA_OPTS not set in production environment

### ISSUE THREAD

**Issue by arnejduranovic** [CDCgov/prime-reportstream#16528]
# DevSecOps Issue

## Summary

#16235 updated Terraform to set the `JAVA_OPTS` env variable with value `-Dfile.encoding=UTF-8` but this variable is not getting set in production. For Staging, @GilmoreA6 manually set it via the Azure Portal web interface.

## Environment

- [ ] Local
- [ ] Dev
- [ ] Stage
- [ ] Prod

## Priority

- [ ] **Critical** - affecting prod systems
- [ ] **Major** - blocking major functionality, deployment, etc
- [ ] **Minor** - improvements, bug fixes
- [ ] **Nice-to-have** - feature request

### Blocks the following

- List any blockers
- Link any Github relevant issues

### Blocked by the following

- Link to any tickets that add context, etc

## Contact

**Team**:

**Slack Team Channel**:

**Slack Contact (Full Name):**

**Technical Team Lead:**

## Definition of Done

- [ ]

## Context Links

**Git Repo:**

**Relevant Code Links:**

**Azure**:

**Site URL**:

**Other**:

## Notes

-

**Comment by devopsmatt** [comment]
This was resolved [here](https://github.com/CDCgov/prime-reportstream/pull/16235)
Any deployment after merging of the branch referenced above should've deployed the updated Terraform config.

**Comment by devopsmatt** [comment]
Blocked by #16819
